install/upgrade/update_to_dev.php

```diff
@@ -199,51 +199,67 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    ];
    $tasksTable = 'glpi_plugin_flyvemdm_tasks';
    $fleetTable = 'glpi_plugin_flyvemdm_fleets';
+   $agentsTable = 'glpi_plugin_flyvemdm_agents';
    $request = [
       'FIELDS'     => [
-         $table => ['symbol'],
-         $tasksTable => ['id', 'itemtype_applied', 'items_id_applied'],
-         $fleetTable => ['entities_id'],
+         $table       => ['symbol'],
+         $tasksTable  => ['id', 'itemtype_applied', 'items_id_applied'],
+         $fleetTable  => ['entities_id'],
       ],
       'FROM'       => $table,
       'INNER JOIN' => [
          $tasksTable => [
             'FKEY' => [
                $tasksTable => 'plugin_flyvemdm_policies_id',
-               $table => 'id'
-            ]
+               $table      => 'id',
+            ],
          ],
-         $fleetTable => [
-            'FKEY' => [
+      ],
+      'LEFT JOIN'  => [
+         $fleetTable  => [
+            'itemtype_applied' => 'PluginFlyvemdmFleet',
+            'FKEY'             => [
                $tasksTable => 'items_id_applied',
                $fleetTable => 'id',
             ],
          ],
+         $agentsTable => [
+            'itemtype_applied' => 'PluginFlyvemdmAgent',
+            'FKEY'             => [
+               $tasksTable  => 'items_id_applied',
+               $agentsTable => 'id',
+            ],
+         ],
+      ],
+      'WHERE'      => [
+         'symbol' => $policies,
       ],
-      'WHERE'      => ['symbol' => $policies],
    ];
    $result = $DB->request($request);
    if (count($result) > 0) {
       $mqttClient = PluginFlyvemdmMqttclient::getInstance();
       foreach ($result as $data) {
+         $type = '';
+         $entities_id = 0;
```

Comment:
dangerous: it may send a mqtt message if  the row does not match none of fleet and agent itemtypes.

Add a ```default``` case in  the switch, with a ```continue``` instruction.